src/base/bittorrent/session.cpp

```diff
@@ -91,7 +91,7 @@
 
 static const char PEER_ID[] = "qB";
 static const char RESUME_FOLDER[] = "BT_backup";
-static const char USER_AGENT[] = "qBittorrent " VERSION;
+static const char USER_AGENT[] = "qBittorrent/" QBT_VERSION_2;
```

Comment:
* We are also changing the whitespace to forward slash, I would guess they (tracker operators) need to add new rules anyway.
  Just to let you know we are joining the minorities... only a small portion have the "v" in version http://www.useragentstring.com/pages/useragentstring.php?name=All

* Regardless of the change, we should mention this on www.qbittorrent.org website beforehand to minimize the impact.